src/singularity.ts

```diff
@@ -1471,7 +1472,32 @@ export const calculateEffectiveSingularities = (singularityCount: number = playe
 
     return effectiveSingularities
 }
+export const calculateNextSpike = (singularityCount: number = player.singularityCount): number => {
+    let PenaltyArray = 11;
+    if (singularityCount > 10) {
+        PenaltyArray = 26
+    }
+    if (singularityCount > 25) {
+        PenaltyArray = 37
+    }
+    if (singularityCount > 36) {
+        PenaltyArray = 51
+    }
+    if (singularityCount > 50) {
+        PenaltyArray = 101
+    }
+    if (singularityCount > 100) {
+        PenaltyArray = 151
+    }
+    if (singularityCount > 150) {
+        PenaltyArray = 250
+    }
+    if (singularityCount === 250) {
+        PenaltyArray = 1337
+    }
 
+    return PenaltyArray
```

Comment:
Again, use it if you like it. Getting around the cap/end at 250. You may want to return -1 outside the loop in case it gets called above array end. It never should, but code changes. And typescript will probably demand it, anyway.

Using the array format makes it easier(?) if more thresholds are added to just have to change fewer parts.

Reply:
Added in this suggestion!